yt_dlp/extractor/radiko.py

```diff
@@ -92,9 +109,18 @@ def _extract_formats(self, video_id, station, is_onair, ft, cursor, auth_token,
         formats = []
         found = set()
         for url_tag in m3u8_urls:
-            pcu = url_tag.find('playlist_create_url')
+            pcu = url_tag.find('playlist_create_url').text
+            if (
+                is_onair and not pcu.startswith(self._HOSTS_FOR_LIVE)
+                # Allowlist style for Time Free to consider undiscovered URL
+                or (not is_onair and (
+                    pcu.startswith(self._HOSTS_FOR_TIME_FREE_FFMPEG_UNSUPPORTED)
+                    or pcu.startswith(self._HOSTS_FOR_LIVE)
+                ))
+            ):
```

Comment:
```suggestion
            if is_onair and not pcu.startswith(self._HOSTS_FOR_LIVE) or (not is_onair and (pcu.startswith(self._HOSTS_FOR_TIME_FREE_FFMPEG_UNSUPPORTED) or pcu.startswith(self._HOSTS_FOR_LIVE)))):
```

`Allowlist style for Time Free to consider undiscovered URL` may be moved above `_HOSTS_FOR_TIME_FREE_FFMPEG_UNSUPPORTED`

Reply:
No problem, I'll leave it to you👍